### ISSUE TITLE
Duplicate values in "Other nutritional substances added:", like Taurine or Inositol

### ISSUE THREAD

**Issue by aleksejrs** [openfoodfacts/openfoodfacts-server#1313]
Some products with Taurine and other substances have them listed multiple times under "Other nutritional substances added:".

Products in https://world.openfoodfacts.org/other-nutritional-substance/taurine :

* https://world.openfoodfacts.org/product/9002490210717/red-bull-energy-drink has 14 pairs of "Taurine" and "Inositol".
* https://world.openfoodfacts.org/product/3041091001344/bledilait-1er-age-bledina (an infant formula, not an energy drink) has a group of 4 ingredients listed twice.
* https://world.openfoodfacts.org/product/0716514229034/lactorelief-iron-fortified-toddler-formula-nature-s-one-inc has 3 ingredients listed only once.

**Comment by hangy** [comment]
Duplicate of #1778
